app/views/govuk_publishing_components/components/docs/chat_entry.yml

```diff
@@ -0,0 +1,37 @@
+name: Chat entry
+description: Entry point for GOV.UK Chat
+accessibility_criteria: |
+  The heading must:
+
+  - be part of a correct heading structure for a page
+  - be semantically represented as a heading
+  - convey the heading level
+uses_component_wrapper_helper: true
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      heading_text: Go to GOV.UK Chat
```

Comment:
The default is to not pass it anything, so perhaps that should be shown here?  Especially as there is an example with a custom heading?

Reply:
Thanks @leenagupte. I see now that components without any required options, like the cookie banner or skip link components, are created in a similar way. I think I'll follow that approach here too.